include/opt-sched/Scheduler/bb_spill.h

```diff
@@ -131,6 +132,7 @@ class BBWithSpill : public SchedRegion {
   // cost of the schedule using Scf whenever the spill cost updates
   void addRecordedCost(SPILL_COST_FUNCTION Scf);
   void storeExtraCost(InstSchedule *sched, SPILL_COST_FUNCTION Scf);
+  InstCount getUnnormalizedIncrementalRPCost();
```

Comment:
```suggestion
  InstCount getUnnormalizedIncrementalRPCost() override;
```

(or `const override`)

Reply:
see the needsSLIL() comment